src/redux/auth/auth-selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { RootState } from './../store';
+const getIsLoggedIn = (state:RootState) => state.auth.isLoggedIn
+
+const getUserEmail = (state:RootState) => state.auth.user.email
+
+const getIsFetchingCurrent = (state:RootState) => state.auth.isFetchingCurrentUser
+
+const getAccessToken = (state:RootState) => state.auth.token
+
+const getSid = (state:RootState) => state.auth.sid
```

Comment:
Please use line breaks correctly. Like this:

import ...

const ...
const ...
...

const all ...

export default